Patch by Robert Scheck <robert@fedoraproject.org> for dislocker >= 0.5.1 to fix destination
paths of the created symlinks; see: https://github.com/Aorimn/dislocker/pull/55

--- dislocker-0.5.1/src/CMakeLists.txt			2016-01-03 16:55:16.000000000 +0100
+++ dislocker-0.5.1/src/CMakeLists.txt.destdir		2016-01-06 02:29:14.283515587 +0100
@@ -229,8 +229,8 @@
 	set (BIN_FIND true)
 endif()
 
-install (CODE "execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${BIN_FUSE} \"${bindir}/${PROJECT_NAME}\")")
-install (CODE "execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${BIN_FUSE}.1.gz \"${mandir}/man1/${PROJECT_NAME}.1.gz\")")
+install (CODE "execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${BIN_FUSE} \"\$ENV{DESTDIR}${bindir}/${PROJECT_NAME}\")")
+install (CODE "execute_process (COMMAND ${CMAKE_COMMAND} -E create_symlink ${BIN_FUSE}.1.gz \"\$ENV{DESTDIR}${mandir}/man1/${PROJECT_NAME}.1.gz\")")
 
 set_directory_properties(PROPERTIES ADDITIONAL_MAKE_CLEAN_FILES "${CLEAN_FILES}")
 
